relint/parse.py

```diff
@@ -124,7 +125,10 @@ def print_culprits(matches, args):
                     ),
                     highlight_lines=range(start_line_no, end_line_no + 1),
                 )
-                message = Group(syntax, hint)
+                if test.hint is None:
```

Comment:
I believe you don't need to include empty string hints either. Therefore, a regular boolean check would probably be better.

```suggestion
                if test.hint:
```

Reply:
Yeah, deciding whether to compare against none or do a boolean check depends on who is maintaining the repository. Changed it.